README.md

```diff
@@ -102,6 +102,67 @@ $ yarn dev
 
 Open http://dev.localhost:3000/ and test away!
 
+## Docker
+```
+```
+
+</p>
+</details>
+
+---
+
+<details><summary><b>Project setup in Docker</b></summary>
+<p>
+
+## Ports used
+
+```
+5500 - Frontend (client)
+5501 - Backend (server)
+5502 - postgres
+5503 - redis
+```
+
+## Setup dofus lab in docker
```

Comment:
Assuming this is intended for local development, are there any instructions for how local development will flow with this setup? Will we have to recreate containers every time we have changes?

Reply:
As i don't play dofus, i play dofus-touch my initial goal was to setup DofusLab locally and import the dofus-touch data and use it that way.

But developers could for example (if developing the client) startup up only postgres / redis / server, and debug the client with their IDE

> Will we have to recreate containers every time we have changes?
Docker does have some live-reload / file watchers but i really didn't test them out, i think it will for sure be slower than without docker the only question is by how much if its 1-2 sec slower its ok, if its 20 sec slower its bad....